src/AutoRest.CSharp/Common/Generation/Writers/JsonCodeWriterExtensions.cs

```diff
@@ -203,9 +203,16 @@ public static void ToSerializeCall(this CodeWriter writer, JsonSerialization ser
                             writer.Line($"{writerName}.WriteObjectValue({name:I});");
                             return;
 
+                        case EnumType clientEnum when clientEnum.IsIntValueType & !clientEnum.IsExtensible:
```

Comment:
what is the best practise here?  use '&&' or '&'

Reply:
Good catch. It should be `&&`. I'll change to use a more explicit way.